### PR TITLE
[Bugfix] Fix encoding

### PR DESCRIPTION
**Pull request by pannga**
Version 1.3.0 enabled encoding in `RequestController` - it was used to fix encoding for both server side and client side SDKs. This introduced issue, as we did not notice that ClientSide get methods were explicitly encoding query params (`toQueryParams` function). That meant the query params were encoded twice and that lead to issues with characters such as %.

### CODE REVIEW

**Comment by alex-all3dp** [comment]
I suppose this fixes https://github.com/voucherifyio/voucherify-js-sdk/issues/94 ? :)

**Comment by pannga** [comment]
> I suppose this fixes https://github.com/voucherifyio/voucherify-js-sdk/issues/94 ? :)

Hey, I'll inform You when the issue is resolved 🚀🚀

File: packages/sdk/src/helpers.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export function assert(condition: any, message?: string): asserts condition {
 	throw new Error(message)
 }
 
-export function toQueryParams(obj: Record<string, unknown>): Record<string, string> {
+export function toQueryParams(obj: Record<string, unknown> | any): Record<string, string> {
```
Comment (darekg11) [inline packages/sdk/src/helpers.ts:53]: Is this change really necessary?

**Review by darekg11 — APPROVED** [review]
Looks good to me, please remember to release both SDK and React-widget

**Comment by darekg11** [comment]
@alex-all3dp Yes, this is meant to fix encoding query params in Client Side SDK.
In version 1.3.0 we have enabled encoding in `RequestController` to fix encoding for both server side and client side SDKs.
Unfortunately we did not notice that Client Side SDK was explicitly encoding query params, because of that for Client Side SDK query params were encoded twice and that lead to issues with characters such as `%`

File: packages/sdk/src/helpers.ts (hunk 1 above)
Comment (pannga) [reply packages/sdk/src/helpers.ts]: Not anymore.
